.travis.yml

```diff
@@ -25,6 +25,8 @@ script:
 - make
 - git diff --exit-code
 - git diff --cached --exit-code
+- mkdir -p tmp
```

Comment:
Kinda.  Some work on another branch that depends on repo settings env vars makes this (or equiv) necessary in order to pass.  Either that or we unset the repo settings env vars in the meantime (?)